### PR TITLE
new UI pictures, add updates tab explanation, switch embassy tab to s…

### PR DESCRIPTION
**Pull request by BitcoinMechanic**
…ystem, remove marketplace switching and settings from system tab

### CODE REVIEW

File: site/source/user-manual/dashboard-overview.rst [hunk 1]
```diff
@@ -16,23 +16,50 @@ Use this tab to view and access your installed services.
     .. figure:: /_static/images/walkthrough/servicestab.png
         :width: 60%
 
-Embassy Tab
+Marketplace Tab
+---------------
+
+Use this tab to access your preferred Marketplace, where you can discover and install new services, or update existing services. Check out the `live Marketplace <https://marketplace.start9.com>`_.
```
Comment (kn0wmad) [inline site/source/user-manual/dashboard-overview.rst:22]: 'Check out a live version of Start9's Official Marketplace on our website'

File: site/source/user-manual/dashboard-overview.rst [hunk 2]
```diff
@@ -47,6 +74,10 @@ The Support section provides links to frequently asked questions as well as Star
 Power
```
Comment (kn0wmad) [inline site/source/user-manual/dashboard-overview.rst:74]: Please add remaining power options - Shutdown and System Rebuild

File: site/source/user-manual/dashboard-overview.rst [hunk 3]
```diff
@@ -16,23 +16,50 @@ Use this tab to view and access your installed services.
...
 
 The Backups section allows you to create and restore :ref:`Embassy backups <backups>`.
 
-Settings
-========
+Manage
+======
 
-The Settings section gives you access to :ref:`embassyOS Updates <updating-eos>`, Device Preferences, :ref:`connecting-lan`, :ref:`ssh`, :ref:`wifi`, and :ref:`alt-marketplaces`.
+The Manage section gives you access to Device Preferences, :ref:`connecting-lan`, :ref:`ssh`, :ref:`wifi`, and Service Sideloading.
```
Comment (kn0wmad) [inline site/source/user-manual/dashboard-overview.rst:62]: Please add OS updates and link to updating page if applicable.

**Review by kn0wmad — CHANGES_REQUESTED** [review]
Finally please add a section, probably near the top, explaining the connection status bar in the bottom left, with some pictures

File: site/source/user-manual/dashboard-overview.rst (hunk 2 above)
Comment (BitcoinMechanic) [reply site/source/user-manual/dashboard-overview.rst:74]: They are there already?

**Review by kn0wmad — APPROVED** [review]

**Pull request merged**